.github/workflows/rust.yml

```diff
@@ -104,11 +87,9 @@ jobs:
           ndk-version: r26b
 
       - name: Install deps
-        uses: actions-rs/cargo@v1
-        with:
-          command: install
-          args: cargo-apk
+        run: cargo install cargo-apk
 
+        # TODO: Do we still need this?
         # Create folder without content to make the build succeed
       - run: |
           mkdir -p deps/android_openxr/arm64-v8a
```

Comment:
The answer is no if we call `prepare-deps`, but not sure that we need to call prepare-deps

Reply:
I'll also try that in the windows image update pr, but I think this one should get finished

Reply:
We do not need prepare deps, we are already installing the android target and cargo-apk for compilation

Reply:
Still, next pr